core/src/main/scala/cmt/CMTcConfig.scala

```diff
@@ -22,7 +22,7 @@ class CMTcConfig(studentifiedRepo: File):
   import Helpers.adaptToOSSeparatorChar
 
   private val cmtConfigFile = studentifiedRepo / ".cmt/.cmt-config"
-  if !cmtConfigFile.exists then printErrorAndExit("missing CMT configuration file")
+  if !cmtConfigFile.exists then printErrorAndExit("Missing CMT configuration file, have you set your current course?")
```

Comment:
Good suggestion; so this happens when one hasn't set used `cmtc set-current-course` to set the current course, or the folder passed to `cmtc set-current-course` isn't pointing to the root folder of a course.